x-pack/plugins/fleet/server/index.ts

```diff
@@ -34,8 +34,7 @@ export const config: PluginConfigDescriptor = {
   deprecations: ({ renameFromRoot, unused }) => [
     renameFromRoot('xpack.ingestManager', 'xpack.fleet'),
     renameFromRoot('xpack.fleet.fleet', 'xpack.fleet.agents'),
-    unused('agents.kibana.ca_sha256'),
-    unused('agents.kibana.host'),
+    unused('agents.kibana'),
```

Comment:
This means all agents.kibana.* configs are deprecated?

Reply:
yes thats correct